### ISSUE TITLE
VersionTally: ArrayIndexOutOfBoundsException when initializing from block store.

### ISSUE THREAD

**Issue by schildbach** [bitcoinj/bitcoinj#1053]
@rnicoll When my app constructs the BlockChain object, it crashes.

```
Caused by: java.lang.ArrayIndexOutOfBoundsException: length=0; index=0
 	at org.bitcoinj.utils.VersionTally.add(VersionTally.java:63)
 	at org.bitcoinj.utils.VersionTally.initialize(VersionTally.java:116)
 	at org.bitcoinj.core.AbstractBlockChain.<init>(AbstractBlockChain.java:160)
 	at org.bitcoinj.core.BlockChain.<init>(BlockChain.java:75)
 	at org.bitcoinj.core.BlockChain.<init>(BlockChain.java:49)
 	at org.bitcoinj.core.BlockChain.<init>(BlockChain.java:55)
 	at de.schildbach.wallet.service.BlockchainServiceImpl.onCreate(BlockchainServiceImpl.java:630)
```

**Comment by schildbach** [comment]
By peeking at the code, it seems majorityWindow is 0 for testnet3. Investigating…

**Comment by schildbach** [comment]
Fixed via bee75f8d369d3f305aa0840202305a1e3c8e6bd0.
